ingest/src/main/java/com/microsoft/azure/kusto/ingest/ManagedStreamingIngestClient.java

```diff
@@ -375,4 +374,14 @@ public void close() throws IOException {
         queuedIngestClient.close();
         streamingIngestClient.close();
     }
+
+    @Override
```

Comment:
Maybe I just need to understand the design better, but I'm not used to seeing both composition (`QueuedIngestClientImpl this.queuedIngestClient`) and inheritance (`implements QueuedIngestClient`) of the such similar classes.

Reply:
https://en.wikipedia.org/wiki/Delegation_pattern

Reply:
Fixed as we said